Commands/Commands.cs

```diff
@@ -23,5 +23,14 @@ public async Task GetLatency(CommandContext ctx)
                 .Build();
             await ctx.Channel.SendMessageAsync(embed: latency);
         }
+        [Command("F")]
+        private async Task F()
+        {
+            await ReplyAsync("F");
+        }
+        [Command("say")]
```

Comment:
We need a role to run this command; the "Member" role.

Reply:
Ok so we need to 'check roles'